app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class=recipe-card>
+  <%= link_to recipe do %>
+  <div class="recipe-container" id="<%= dom_id recipe %>">
+      <div class="recipe-flex">
+      <p class="recipe-name">
+        <strong>Name:</strong>
+        <%= recipe.name %>
+      </p>
+    </div>
+    <div class="recipe-flex">
+      <p>
+        <strong>Description:</strong>
+        <%= recipe.description %>
+      </p>
+
+      <p>
+        <strong>Public:</strong>
+        <%= recipe.public %>
+      </p>
+    </div>
+  </div>
+  <% end %>
+  <%= button_to "Remove", @recipes_path, method: :delete, data: { confirm: "Are you sure?" }, class: "actions" %>
+</div>
```

Comment:
- [ ] In the "Recipies list" page,  The delete button is still dysfunctional. When I click on the "delete" button, the page reloads without any action. Kindly ensure that the owner can remove the recipes from the Recipes list page.
- [ ] In the "Recipe details" page, clicking on the recipe button shows the error below. When a recipe contains food, the user cannot delete it. To solve that, I recommend reading [this conversation](https://stackoverflow.com/questions/48739646/pgforeignkeyviolation-error-update-or-delete-on-table-xxx-violates-foreign) in StackOverflow.

<img width="909" alt="image" src="https://user-images.githubusercontent.com/77190203/203138870-ee6bbdaa-4438-4e10-b850-27b2de6bef08.png">